Automattic/woocommerce-payments
# Adjust style of save my info based on previous section

Fixes #6640.

#### Changes proposed in this Pull Request

It makes the "Save my info" section follow the style of the "Payment options" in the Blocks checkout when it comes to contain or not the step number. Since this section is not registered in Blocks, it's not configurable, so it's currently always displaying the step number.








<table>
<tr>
<th>Before: Payment options with step number disabled</th>
<th>Now: Payment options with step number enabled</th>
<th>Now: Payment options with step number disabled</th>
</tr>
<tr>
<td>
<img width="500" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/aefb2564-d6df-4c5b-97f9-fa7f03bd3d2d">
</td>
<td>
<img width="519" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/a2d2055c-aa00-4d65-a5ce-7b78f0c08d48">
</td>
<td>
<img width="525" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/c02ac2d4-1444-4be1-b487-0857e50881a7">
</td>
</tr>
</table>

#### Testing instructions





1. Go to WP Admin > Pages and edit the Blocks checkout page.
2. Click on the "Payment options" section, and in sidebar's "Block" tab, toggle the "Show step number" option and save.
3. Confirm that:
3.1. When enabled, the checkout looks like the second screenshot above.
3.2. When disabled, the checkout looks like the third screenshot above.

-------------------


**Post merge**
